CI-CD/CircleCI/circleci-pipeline-npm-scan

```diff
@@ -20,8 +20,6 @@ jobs:
       # WS_WSS_URL
       # WS_PRODUCTNAME
       # WS_PROJECTNAME
-      # WS_RESOLVEALLDEPENDENCIES: false
-      # WS_NPM_RESOLVEDEPENDENCIES: true
```

Comment:
If removed - filename should be changed as well

Reply:
as discussed otp